examples/custom-action/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "custom-action-example",
+  "version": "1.0.0",
+  "description": "",
+  "main": "build/index.js",
+  "scripts": {
+    "pretest": "npm run build",
+    "test": "mocha build/tests/*.js --exit",
+    "build": "tsc"
+  },
+  "author": "",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@mediarithmics/plugins-nodejs-sdk": "^0.7.9"
```

Comment:
Custom actions are not supported in the version 0.7.9 of the SDK, aren't they?

From experience, we need to do 2 PRs:
1. First one to add the support of the custom actions in the SDK and do a release of a new version
2. Second one to release the example plugin based on the new version released after 1.

In time, in order to make this process more intuitive, we can separate the repo of the SDK and of the examples in 2 separate repos.